server/server/models/user_status.py

```diff
@@ -14,3 +14,9 @@ class UserStatus(Base):
     status = Column(String)
 
     users = relationship('User')
+
+    @classmethod
+    def get_status_id(cls, request, status):
```

Comment:
This method return an object of UserStatus model, not also an id, so the name `get_status_id` is not suitable for this method.

Reply:
I know it, but for me it's suitable, cause I call this method when I set status_id into the table users, this way I created this name for simple understanding for you.